Resources/Prototypes/Roles/Jobs/Science/research_director.yml

```diff
@@ -29,6 +29,7 @@
     components:
     - type: BibleUser # Nyano - Lets them heal with bibles
     - type: Psionic # Nyano - They start with telepathic chat
+      powerSlots: 3
```

Comment:
Don't forget his ifrit

Reply:
Ifrit can't actually roll powers, so no need. It's defaulted to 1, and the ifrit has 1 power.